integration/int_3Ri_2BhaRi2_3Re_2BhaRe3/tasks/pip_modules.yml

```diff
@@ -4,4 +4,4 @@
     name:
       - docker
       - selinux
-    extra_args: --user
+    #extra_args: --user
```

Comment:
If we do not need it, I would remove the line instead of commenting. Moreover that, approved.

Reply:
done.